src/ir/function.rs

```diff
@@ -635,6 +635,12 @@ impl ClangSubItemParser for Function {
             // but seems easy enough to handle it here.
             name.push_str("_destructor");
         }
+        if let Some(callbacks) = context.parse_callbacks() {
+            if let Some(nm) = callbacks.generated_name_override(&name) {
+                name = nm;
+            }
+        }
+        assert!(!name.is_empty(), "Empty function name.");
```

Comment:
Not a blocker for this PR but in the future we might want to change `ParseResult` to float up this kind of errors nicely.